tests/run_tests.py

```diff
@@ -292,7 +296,7 @@ def run_system(config_file, vic_exe, test_data_dir, out_dir, driver):
 
         # ------------------------------------------------------------ #
         # read template global parameter file
-        infile = os.path.join(test_dir, 'samples',
+        infile = os.path.join(test_data_dir + 'image/Stehekin/parameters/', 
```

Comment:
If we assume that the global parameter template file is under a certain directory, it should probably be put under `system` subdirectory under `test_dir`, so `infile = os.path.join(test_dir, 'system', test_dict['global_parameter_file'])`